_includes/post-paginator.html

```diff
@@ -47,7 +47,7 @@
     {% if show %}
       <!-- show number -->
       <li class="page-item {% if i == paginator.page %} active{% endif %}">
-        <a class="page-link btn-box-shadow" href="{{ site.baseurl }}/{% if i > 1%}page{{ i }}/{% endif %}">{{ i }}</a>
+        <a class="page-link btn-box-shadow" href="{% if i > 1 %}{{ site.paginate_path | replace: ':num', i | relative_url }}{% else %}{{ '/' | relative_url }}{% endif %}">{{ i }}</a>
```

Comment:
I don't think `/page{num}/` is an illegal path, please undo the changes here

Reply:
/page{num}/ does not work if "paginate_path" in _config.yml is changed to "/page/:num/".

Reply:
Is there any particular reason to set it to `/page:num/` in _\_config.yml_? The default one `/page:num/` is good enough.

Reply:
I used to use Wordpress which had the pages as `/page/x/` so to keep links and things working I also want to use the same paths in Jekyll.

Reply:
And of course if someone else wants to change the page paths for whatever reason then it will just work...

Reply:
Alright, got it